app/assets/main/react_components/dashboard/climate_actions/carousel/CarouselList.jsx

```diff
@@ -34,21 +33,25 @@ const CarouselList = ({ user, updateLocalAccepted, categories }) => {
 
   const actions = isMobile ? sortForMobileClimateActions : climateActions;
 
+  const showNoOfActions = isMobile
+    ? 1.5
+    : isLargeTablet
+    ? 2.5
+    : isDesktop
+    ? 3
+    : 4;
+
+  const loopActions = actions.length > showNoOfActions ? true : false;
+  const noArrows = isMobile || !loopActions ? true : false;
```

Comment:
A bit unclear if 'no' stands for 'no' or 'number' since 'no' is used for 'number ' in the previous row. Maybe use 'showArrows' instead (and if so, change the condition to suit the name)?